src/Seq.App.EmailPlus/EmailApp.cs

```diff
@@ -132,18 +145,15 @@ public void On(Event<LogEventData> evt)
 
             var to = FormatTemplate(_toAddressesTemplate.Value, evt, base.Host);
             var body = FormatTemplate(_bodyTemplate.Value, evt, base.Host);
-            var subject = FormatTemplate(_subjectTemplate.Value, evt, base.Host).Trim().Replace("\r", "").Replace("\n", "");
+            var subject = FormatTemplate(_subjectTemplate.Value, evt, base.Host).Trim().Replace("\r", "")
+                .Replace("\n", "");
             if (subject.Length > MaxSubjectLength)
                 subject = subject.Substring(0, MaxSubjectLength);
 
-            var client = new SmtpClient(Host, Port ?? 25) {EnableSsl = EnableSsl ?? false};
-            if (!string.IsNullOrWhiteSpace(Username))
-                client.Credentials = new NetworkCredential(Username, Password);
+            var result = await _mailGateway.Send(_client, _options, new MimeMessage(new List<InternetAddress> {MailboxAddress.Parse(From)},
```

Comment:
Since `result` is unused, we'll no longer get errors logged to the app's diagnostic stream when sending fails. Perhaps until we've looked into the fallback server idea, we should just keep throwing exceptions on failure, here?

Reply:
Agree. I was planning to log errors as part of this but didn't implement a throw in the interim

Reply:
Updated - if !result.Success, we throw result.Errors.